tools/accuracy_checker/accuracy_checker/postprocessor/resize_segmentation_mask.py

```diff
@@ -79,3 +80,52 @@ def _(entry, height, width):
             resize_segmentation_mask(target, target_height, target_width)
 
         return annotation, prediction
+
+    def to_image(self, arr):
+        data = np.asarray(arr)
+        if np.iscomplexobj(data):
+            raise ValueError("Cannot convert a complex-valued array.")
+        shape = list(data.shape)
+        if len(shape) == 2:
+            return self._process_2d(data, shape)
+        if len(shape) == 3 and ((3 in shape) or (4 in shape)):
```

Comment:
Since we now know which dimension is the channel dimension, `((3 in shape) or (4 in shape))` should be replaced by `shape[2] in {3, 4}`.